org.civicrm.bootstrapcivicrm/scss/civicrm/vacancies/_vacancies.scss

```diff
@@ -139,6 +139,20 @@
       .crm-form-time {
         margin-left: 15px !important;
       }
+
+      tr {
+        td {
+          vertical-align: middle !important;
+        }
+
+        &:nth-child(4),
+        &:nth-child(5),
+        &:nth-child(6) {
```

Comment:
wouldn't it be better to replace those with: `:nth-child(n+4):nth-child(-n+6)` (untested) ?